generator/AWSPSGeneratorLib/HelpMaterials/Examples/Common/Set-AWSProxy.xml

```diff
@@ -14,7 +14,7 @@
   </example>
   <example>
     <code>Set-AWSProxy -Hostname https://proxy.testserver.com -Port 80 -BypassList 169.254.169.250;169.254.169.251;169.254.169.254</code>
-    <description>This command configures a proxy at https://proxy.testserver.com that uses port number 80. The -BypassList parameter is added to show that IP addresses 169.254.169.250, 169.254.169.251, and 169.254.169.254 do not use the proxy. Note that because your bypass list includes only IP addresses and not FQDNs, quotation marks around the IP address values are not needed. You can use semicolons as a delimiter between IP addresses that you want to bypass.</description>
+    <description>This command configures a proxy at https://proxy.testserver.com that uses port number 80. The -BypassList parameter is added to show that IP addresses 169.254.169.250, 169.254.169.251, and 169.254.169.254 do not use the proxy. Note that because your bypass list includes only IP addresses and not FQDNs, quotation marks around the IP address values are not needed. You can use semicolons as a delimiter between IP addresses that you want to bypass. Also, note that if the Windows OS is configured to use IPv4, these IPv4 link-local addresses can be used. If the Windows OS has the IPv4 network protocol stack disabled and uses IPv6 instead, add [fd00:ec2::240] in place of 169.254.169.250 and 169.254.169.251. Then add [fd00:ec2::254] in place of 169.254.169.254.</description>
```

Comment:
I think the new guidance may be at odds with the existing blurb around quotation marks: `quotation marks around the IP address values are not needed`

Multiple IPv6 addresses without quotes fails:
```
> Set-AWSProxy -Hostname https://proxy.testserver.com -Port 80 -BypassList [fd00:ec2::240];[fd00:ec2::254]
At line:1 char:94
+ ... roxy.testserver.com -Port 80 -BypassList [fd00:ec2::240];[fd00:ec2::2 ...
+                                                                  ~
Missing ] at end of attribute or type literal.
At line:1 char:95
```

Wondering if it'd be better to add new `<example>` block with an IPv6-only example?